xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -72,6 +74,7 @@
 #include "cores/AudioEngine/Interfaces/AE.h"
 #include "cores/AudioEngine/Sinks/AESinkAUDIOTRACK.h"
 #include "cores/VideoPlayer/VideoRenderers/RenderManager.h"
+#include "dialogs/GUIDialogKaiToast.h"
```

Comment:
Doh! Making CXBMCApp dependent on high level gui components seems not correct. Usually, XBMCApp interacts with the upper layers of Kodi using messages.

CXBMCApp should only provide low level functionality, not have any GUI.